test/packages/with-custom-agent/apache/_dev/build/docs/README.md

```diff
@@ -0,0 +1,17 @@
+# Apache Integration
```

Comment:
Nit. The apache integration doesn't actually need this deployer. It'd be nice to have a test package that illustrates a real use case where the custom agent would be needed.